package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "github-notifier-chrome",
+  "version": "2.0.0",
+  "description": "Chrome/Opera extension - Displays your GitHub notifications unread count",
+  "main": "index.js",
+  "scripts": {
+    "lint": "eslint extension/*.js test/*.js",
+    "test": "mocha"
+  },
+  "repository": {
+    "type": "git",
+    "url": "sindresorhus/github-notifier-chrome"
+  },
+  "keywords": [
+    "github",
+    "notifier",
+    "chrome",
+    "extension"
+  ],
+  "author": {
+    "name": "Sindre Sorhus",
+    "email": "sindresorhus@gmail.com",
+    "url": "sindresorhus.com"
+  },
+  "license": "MIT",
+  "devDependencies": {
+    "chrome-stub": "^1.2.0",
+    "eslint": "^0.20.0",
+    "istanbul": "^0.3.13",
+    "mocha": "^2.2.4",
+    "mock-localstorage": "^0.1.3",
+    "sinon": "^1.15.4"
+  }
+}
```

Comment:
No point in filling all of this out. It's not going on npm anyways. It's enough with `devDependencies` and `scripts`.